tenable/io/exports.py

```diff
@@ -12,15 +12,17 @@
 
     .. automethod:: assets
     .. automethod:: vulns
+    .. automethod:: compliance
 '''
-from .base import TIOEndpoint, APIResultsIterator, UnexpectedValueError
-from tenable.errors import TioExportsError, TioExportsTimeout
+import time
+import sys
 from ipaddress import IPv4Network
+from tenable.errors import TioExportsError, TioExportsTimeout
+from .base import TIOEndpoint, APIResultsIterator, UnexpectedValueError
```

Comment:
Why cant import served as tenable.io.base

Reply:
refactored import to use full path